tilt-setup/deploy_tilt_env.sh

```diff
@@ -85,17 +99,30 @@ spec:
           - mountPath: /opt/metal3/auth/ironic
             name: ironic-credentials
             readOnly: true
+EOF
+
+if [ -n "${IRONICINSPECTOR_SECRET_NAME}" ]; then
```

Comment:
```suggestion
if [[ -n "${IRONICINSPECTOR_SECRET_NAME}" ]]; then
```
Let's not add new `[ .. ]` since we're using Bash.